src/test/java/org/jabref/model/FieldChangeTest.java

```diff
@@ -0,0 +1,63 @@
+package org.jabref.model;
+
+import org.jabref.model.entry.BibEntry;
+import org.jabref.model.entry.field.Field;
+import org.jabref.model.entry.field.StandardField;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class FieldChangeTest {
+
+  private FieldChange Fieldchange;
+
+  @Test
+  void testHashCode() {
+    FieldChange fcNull = new FieldChange(null, null, null, null);
+    assertEquals(923521, fcNull.hashCode());
+  }
```

Comment:
Please no test for `hashCode`. 

You can include https://jqno.nl/equalsverifier/ as dependency and work on that. Please do not test for that manually.